src/tests/fixtures/enrcp

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
This might also be unneeded on our test image due to Enstore package installation, but I'm not sure